Tests/Functional/Domain/Repository/Product/TeaRepositoryTest.php

```diff
@@ -133,4 +134,53 @@ public function addAndPersistAllCreatesNewRecord(): void
         self::assertIsArray($databaseRow);
         self::assertSame($title, $databaseRow['title']);
     }
+
+    /**
+     * @test
+     */
+    public function findByOwnerUidFindsEventWithTheGivenOwnerUid(): void
+    {
+        $this->importCSVDataSet(__DIR__ . '/../Fixtures/Product/TeaWithOwner.csv');
+
+        $result = $this->subject->findByOwnerUid(1);
+
+        self::assertCount(1, $result);
+    }
+
+    /**
+     * @test
+     */
+    public function findByOwnerUidFindsIgnoresEventWithOtherOwnerUid(): void
```

Comment:
I guess "Other" is too generic. Maybe something like "findByOwnerUidFindsNothingForNoneExistingOwnerUid"? or "findByOwnerUidFindsNothingForNoneMatchingOwnerUid"

Reply:
Done.